library/core/src/lib.rs

```diff
@@ -167,6 +167,7 @@
 #![feature(no_coverage)] // rust-lang/rust#84605
 #![cfg_attr(bootstrap, feature(target_feature_11))]
 #![deny(unsafe_op_in_unsafe_fn)]
+#![allow(renamed_and_removed_lints)]
```

Comment:
Should I add a `#[cfg_attr(not(bootstrap))]` to this?

Reply:
Maybe this would be better:

```rust
#![cfg_attr(bootstrap, deny(or_patterns_back_compat))]
#![cfg_attr(not(bootstrap), deny(rust_2021_incompatible_or_patterns))]
```
to use the new name when available.